articles/fin-ops-core/dev-itpro/lifecycle-services/subscription-estimator.md

```diff
@@ -3,8 +3,8 @@
 
 title: Subscription estimator in Lifecycle Services (LCS)
 description: This topic explains how to use the Subscription estimator tool that is available in Lifecycle Services (LCS).
-author: angelmarshall
-ms.date: 03/25/2021
+author: ryan-mcculloch
```

Comment:
is it appropriate to update the author to be myself (as I am the latest person to change it) or is this intended to be the long term owner of this document?

Reply:
The author should be the person who "owns" this content area, usually the PM for a feature.